wa_lap_identification.sql

```diff
@@ -0,0 +1,240 @@
+/***********************************************************************************************************************
+This query produces a roster of currently enrolled students at Atlas, Olympus, and Sierra and indicates whether they are LAP
+eligible. LAP eligibility is based on meeting the following criteria:
+
+-------
+ 
+LAP MATH:
+
+Bottom 20th percentile  (<= 20)on MAP Math in most recent test administration for that student
+(considering current and previous school year; if student took test more than once in most recent administration, use highest score)
+
+AND
+
+<= C+ in Math in the previous year
+
+-------
+
+LAP ELA:
+
+Bottom 20th percentile (<= 20) on MAP Reading in most recent test administration for that student
+(considering current and previous school year; if student took test more than once in most recent administration, use highest score)
+
+AND
+
+<= C+ in English in the previous year
+
+-------
+
+To update this query for a new school year, update the following:
+ - first day of school
+ - Fall snapshot date
+ - NWEA tables
+ - academic year in the grades tables
+ **********************************************************************************************************************/
+
+SELECT
+  student_set.site_name
+  , student_set.local_student_id
+  , last_name
+  , first_name
+  , student_set.grade_level
+  , map_math.map_percentile AS min_map_math_percentile
+  , map_math.map_term AS min_map_math_term
+  , math_grade
+  , CASE WHEN (map_math.map_percentile <= 20 AND math_grade IN ('C+', 'C', 'C-', 'D+', 'D', 'D-', 'F', 'INCOMPLETE')) THEN TRUE
+         ELSE FALSE
+    END AS lap_math
+  , map_reading.map_percentile AS min_map_reading_percentile
+  , map_reading.map_term AS min_map_reading_term
+  , ela_grade
+  , CASE WHEN (map_reading.map_percentile <= 20 AND ela_grade IN ('C+', 'C', 'C-', 'D+', 'D', 'D-', 'F', 'INCOMPLETE')) THEN TRUE
+         ELSE FALSE
+    END AS lap_ela
+
+
+FROM
+  /* Student set: students enrolled at Atlas, Olympus, and Sierra on 10/01 */
+  (SELECT
+    site_name
+    , students.student_id
+    , local_student_id
+    , last_name
+    , first_name
+    , grade_level_id - 1 AS grade_level
+
+  FROM public.student_session_aff AS enrollments
+
+  LEFT JOIN public.students ON enrollments.student_id = students.student_id
+  LEFT JOIN public.sessions ON enrollments.session_id = sessions.session_id
+  LEFT JOIN public.sites ON sessions.site_id = sites.site_id
+
+  WHERE
+    enrollments.entry_date >= DATE '2018-08-21' --their enrollment was after the first day of school for this SY
+    AND enrollments.entry_date <= DATE '2018-10-01' -- enrollment was on or before the snapshot date
```

Comment:
I think we had discussed 11/1 since that would be after fall MAP testing.  After you update the date feel free to merge the pull request!  Nice work!